feat(project debug): add Go debugging configuration for debugging the project

amkarkhi opened the pull request:
when debugging the project it feels more comfortable to have this config. i thought i might add this for everyone to use

Commit:
feat(project debug): add Go debugging configuration to debug the project

antoineco commented:
It's unlikely to apply universally. A lot of Go projects use the convention of having `main` packages in `cmd/`.

Commit:
fix: add folder selector and args...

amkarkhi commented:
@antoineco i have changed the function. now you can select the folder eg. `cmd/` or `./` so it will fit the use-case you mentioned. also i have added args...
